genetic_algorithms/solvers/local_search_solver.py

```diff
@@ -11,7 +11,7 @@ class LocalSearchSolver(Solver):
     def solve(self, model: Model, algorithm: NextStateProvider) -> State:
         self.start_timer()
         solution = model.initial_solution
-        while not self._is_optimal_solution_found() and not self.timeout():
+        while not self._is_optimal_solution_found():
```

Comment:
Wyrzuciłem dodatkowy warunek żeby dodać informację o przekroczeniu czasu na ekranie, a przez not` self.timeout()` algorytm się za wcześnie kończył i nie pokazywał informacji.  
Czy ` self.timeout()`  był konieczny i czy takie podejście jest ok?
Teraz to wygląda tak:
![image](https://user-images.githubusercontent.com/84449847/134809428-c9beb390-8f89-4a2e-8040-d81b551a62a6.png)